models/category.js

```diff
@@ -6,11 +6,11 @@ const categorySchema = new Schema(
     name: {
       type: String,
       unique: true,
-      required: true
+      required: [true, 'Please, enter a category name']
```

Comment:
gotta think about this message, either make it generic, or universal, coz in other models it is 'this field cannot be empty'
but if it is a generic, then makes sense to move it to const? or something? what do you think?

Reply:
agree with you

Reply:
i have added error to consts, i would be glad if you check @wonderflu